tests/integration/run_tests.py

```diff
@@ -1081,6 +1160,16 @@ async def main():
         LOG.debug("=== Testing basic CRUD user operations ===")
         await test_crud_users_works(sess)
 
+    # Remove the remaining user in the test database
+    async with aiohttp.ClientSession() as sess:
+        await login(sess, other_test_user, other_test_user_given, other_test_user_family)
+        async with sess.get(f"{users_url}/{user_id}") as resp:
+            LOG.debug(f"Reading user {user_id}")
+            assert resp.status == 200, "HTTP Status code error"
+            response = await resp.json()
+            real_user_id = response["userId"]
+        await delete_user(sess, real_user_id)
+
```

Comment:
Adding this allowed me to rerun integration tests multiple times but is not a good long-term solution for dropping database/collection after integration tests have completed.

Reply:
i think for completing this issue will have to do let us create a new issue to clean integration tests.

Reply:
https://github.com/CSCfi/metadata-submitter/issues/234